Added Todo

f-dub-e opened the pull request.

FrancoisFleetZero committed:
Added Todo

fcol95 commented:
Please follow proper PR titling good practices ;)

f-dub-e committed:
latest commit

felixFleetZero commented on fliip_register_class.py:
```diff
@@ -108,6 +108,17 @@ def register_noon_weekday_class(
+    # Wait until the minutes of the hour are equal to the variable "wait_until_minutes"
+    print_in_while_loop = True
+    wait_until_minutes = 20
+    while True:
```
@f-dub-e using a "while True" with an "if X break" for a wait is very inefficient for your computer, especially that you don't have any "down time" in the loop, just doing as fast as the OS allows you (so its using a whole lot of processing power for nothing and slowing down your OS/PC).
A more efficient and elegant way would be to use the "time.sleep(x)" method built-in python.
Here's an example of usage, where the target is a specific datetime, in our case here, you could have a dictionary that gives the hours and minutes of start of each of the class in "noon_classes_to_register":
```
import datetime
import time

# Set the target date and time
target_datetime = datetime.datetime(2024, 11, 20, 12, 0, 0)  # YYYY, MM, DD, HH, MM, SS

# Get the current time
now = datetime.datetime.now()

# Calculate the time difference
time_difference = (target_datetime - now).total_seconds()

if time_difference > 0:
    print(f"Waiting until {target_datetime}...")
    # Sleep until the target date and time
    time.sleep(time_difference)

# After waiting, execute the code
print("The wait is over!")
```

f-dub-e committed:
Changed Monday class id to Aerobic Class Id
Changed name of first dictionary to classes_to_register which includes the start time of the class

Modified code so that if the codes runs before the start of the class in 2 weeks, it will time.sleep and then resume at the time of the class +1ms to click on the register_button class.

f-dub-e committed:
Removed commented datetime
headless is now = True
